unpackaged/config/qa/language/settings/package.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
To clarify, here's what I'm seeing on c57191bb:
```
unpackaged/config/qa
├── applications
├── language
│   ├── settings
│   │   └── settings
│   └── translations
│       └── translations
├── layouts
├── objects
├── profiles
└── tabs
```

Remove those so that you end up with:
```
unpackaged/config/qa
├── applications
├── layouts
├── objects
├── profiles
└── tabs
```